app/src/main/java/ru/suslovalex/androidacademyapp/repository/Repository.kt

```diff
@@ -0,0 +1,11 @@
+package ru.suslovalex.androidacademyapp.repository
+
+import ru.suslovalex.androidacademyapp.data.MoviesResponse
+import ru.suslovalex.androidacademyapp.retrofit.RetrofitInstance
+
+class Repository {
+
```

Comment:
@AleksandrAlex You need to add dataStore and all the mapings here. Please check the example https://gist.github.com/alexander-smityuk/9e3528677c03aaad7b3949fe6a2b74f8 and make changes according to your code. Also I do not see any actors request, please add it.